app/access_grant_funding/routes/runner.py

```diff
@@ -145,3 +156,28 @@ def check_your_answers(
     return render_template(
         "access_grant_funding/reports/check_your_answers.html", grant_recipient=grant_recipient, runner=runner
     )
+
+
+@access_grant_funding_blueprint.route(
+    "/organisation/<uuid:organisation_id>/grants/<uuid:grant_id>/reports/<uuid:submission_id>/confirmation",
+    methods=["GET", "POST"],
+)
+@auto_commit_after_request
```

Comment:
```suggestion
```

Reply:
Tests don't catch it if you _also_ tell it it needs POST 🤦 tyty